app/thankyou/page.tsx

```diff
@@ -1,7 +1,12 @@
+import { Metadata } from "next";
 import ThankyouPage from ".";
 
 export const dynamicParams = false;
 
+export const metadata: Metadata = {
+  title: "Thank you for consulting with SSW",
```

Comment:
"Consulting with SSW" feels a bit weird. We're the ones doing the consulting.


```suggestion
  title: "Thank you for working with SSW",
```